src/core/tests/frontend/paddle/test_models/gen_scripts/generate_reverse.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+#
+# reverse paddle model generator
+#
+import numpy as np
+from save_model import saveModel
+import sys
+
+def reverse(name: str, x, axis, dtype="float32"):
+    import paddle
+    paddle.enable_static()
+
+    with paddle.static.program_guard(paddle.static.Program(), paddle.static.Program()):
+        node_x = paddle.static.data(name='x', shape=x.shape, dtype=dtype)
+        out = paddle.fluid.layers.reverse(node_x, axis)
```

Comment:
can we avoid use the legacy fluid api? Better use paddle.reverse, m.b.

Reply:
> can we avoid use the legacy fluid api? Better use paddle.reverse, m.b.
sure. have fixed it.

Reply:
I am afraid not.
The reason is that paddle.reverse api is always translated to flip instead of reverse.
BTW, from the function view, flip is a part of reverse but OV PD FE hasn't mapped it.
So, the API of paddle.fluid.layers.reverse makes sense here.